src/cockpit/389-console/src/lib/database/referrals.jsx

```diff
@@ -228,22 +245,35 @@ export class SuffixReferrals extends React.Component {
                     rows={this.props.rows}
                     deleteRef={this.showConfirmRefDelete}
                 />
-                <button className="btn btn-primary ds-margin-top" onClick={this.showRefModal} type="button">Create Referral</button>
-                <ConfirmPopup
+                <Button className="ds-margin-top" variant="primary" onClick={this.showRefModal}>
+                    Create Referral
+                </Button>
+                <DoubleConfirmModal
                     showModal={this.state.showConfirmRefDelete}
                     closeHandler={this.closeConfirmRefDelete}
-                    actionFunc={this.deleteRef}
-                    actionParam={this.state.removeRef}
-                    msg="Are you sure you want to delete this referral?"
-                    msgContent={this.state.removeRef}
+                    handleChange={this.handleChange}
+                    actionHandler={this.deleteRef}
+                    spinning={this.state.modalSpinning}
+                    item={this.state.removeRef}
+                    checked={this.state.modalChecked}
+                    mTitle="Delete Refferal"
+                    mMsg="Are you sure you want to delete this referral?"
```

Comment:
While trying to delete the referral, I get the error when I click on 'Yes I am sure':

```
Uncaught TypeError: a is not a function
    at Object.onChange (index.js:72)
    at handleChange (index.js:72)
    at Object.s (index.js:51)
    at f (index.js:51)
    at index.js:51
    at y (index.js:51)
    at it (index.js:51)
    at Array.forEach (<anonymous>)
    at rt (index.js:51)
    at st (index.js:51)
```

Reply:
Fixed